src/main/java/org/jboss/forge/arquillian/observers/DownloadServerObserver.java

```diff
@@ -1,96 +0,0 @@
-/*
- * Copyright 2013 Red Hat, Inc. and/or its affiliates.
- *
- * Licensed under the Eclipse Public License version 1.0, available at
- * http://www.eclipse.org/legal/epl-v10.html
- */
-package org.jboss.forge.arquillian.observers;
-
-import org.jboss.forge.addon.projects.Project;
-import org.jboss.forge.addon.ui.input.UIPrompt;
-
-import javax.inject.Inject;
-
-/**
- * @Author Paul Bakker - paul.bakker.nl@gmail.com
- */
-public class DownloadServerObserver {
```

Comment:
1.In this we were creating dynamic input depending on user input for installContainer. I observered that there is prompt available which is attached to  there but can't give description there.
2. While creating dynamic inputs for version selection, we can't get that dynamic inputs in interactive mode.
3. All this logic is related to MavenProfile So I think it's good to move it into `ProfileManager` as per SRP.